Water_Cluster-SCF_Guess/compile_water_cluster_record.py

```diff
@@ -0,0 +1,44 @@
+from wrap_pyscf import *
+
+import pickle
+
+scf_guess = {}
+
+filename = "scf_record"
+
```

Comment:
Same comments as the other file, but I wouldn't consider them as imperative until we come to grips with what we are going to do for the guess.